packages/itwinui-react/src/core/InformationPanel/InformationPanel.tsx

```diff
@@ -56,12 +56,36 @@ export const InformationPanel = React.forwardRef((props, ref) => {
     isOpen = false,
     orientation = 'vertical',
     resizable = true,
-    children,
     ...rest
   } = props;
 
+  return (
+    <InformationPanelInternal
```

Comment:
It would be better to pass all props directly to `InformationPanelInternal` instead of handling some of them here.

```jsx
<InformationPanelInternal
  key={props.orientation}
  ref={forwardedRef}
  {...props}
/>
```

Reply:
Okay, will do.

Reply:
Done in this [commit.](https://github.com/iTwin/iTwinUI/pull/1964/commits/7b041b9609f61f2c90fac1398436da887c49f3e6)